runtime/parallel/src/lib.rs

```diff
@@ -1773,12 +1773,14 @@ impl pallet_crowdloans::Config for Runtime {
 
 parameter_types! {
     pub const StreamPalletId: PalletId = PalletId(*b"par/strm");
+    pub const MaxStreamsCount: u32 = 32;
```

Comment:
Not sure if it's true. If the user is an account in a company, he wants to send the streams for their employee. I believe there will be more than 32 streams?

Reply:
Maybe we need to discuss it.

Reply:
this value is easy to change, we can adjust in the future, but for now I think 32 is very enough actually

Reply:
128 Should be good to start. I think we will have more than 32 from our account. Confirmed with @Helen8

Reply:
okay